Add two operators in AWS Providers: RedshiftResumeClusterOperator and RedshiftPauseClusterOperator (#19665)
mistercrunch/Airflow: docs/apache-airflow-providers-amazon/operators/redshift.rst

    :language: python
    :start-after: [START redshift_operator_howto_guide]
    :end-before: [END redshift_operator_howto_guide]

.. _howto/operator:RedshiftResumeClusterOperator:

Resume a Redshift Cluster
"""""""""""""""""""""""""""""""""""""""""""

> To resume an existing AWS Redshift Cluster you can use

Reviewer: ```suggestion
To resume a 'paused' AWS Redshift Cluster you can use
```

:class:`~airflow.providers.amazon.aws.operators.redshift_resume_cluster.RedshiftResumeClusterOperator`.

This Operator leverages the AWS CLI
`resume-cluster <https://docs.aws.amazon.com/cli/latest/reference/redshift/resume-cluster.html>`__ API

.. _howto/operator:RedshiftPauseClusterOperator:

Pause a Redshift Cluster
"""""""""""""""""""""""""""""""""""""""""""

> To pause an existing AWS Redshift Cluster you can use

Reviewer: ```suggestion
To pause an 'available' AWS Redshift Cluster you can use
```
